AG-469_config.yaml

```diff
@@ -0,0 +1,236 @@
+- destination: &dest syn38973163
```

Comment:
Both test_config.yaml and config.yaml have been updated with all the config changes. (commit [9b9545112e0adbbbf94e37a79a43814a12a4b690](https://github.com/jaclynbeck-sage/agora-data-tools/commit/9b9545112e0adbbbf94e37a79a43814a12a4b690).

Per our meeting with Anna last week, I've also over-written the old feather file with the new feather file on Synapse, so the current version of syn25953363 is now the updated gene list. test_config.yaml and config.yaml both point to this synapse ID.

Reply:
Excellent! 

Since you've updated the other config files now, do you think we still need this extra one? Unless there's a reason to, I'd rather not add it to the repo.